x/emissions/module/rewards_calc.go

```diff
@@ -300,6 +291,8 @@ func emitRewardsToTopicParticipants(
 	for participant, reward := range rewards {
 		fmt.Printf(" Emitting %suallo to %s \n", reward.String(), participant)
 		am.keeper.AddStake(ctx, []uint64{topic}, participant, participant, *reward)
+		rewardCoins := sdk.NewCoins(sdk.NewCoin(params.DefaultBondDenom, cosmosMath.NewIntFromBigInt(reward.BigInt())))
+		am.keeper.BankKeeper().SendCoinsFromModuleToModule(ctx, state.AlloraRewardsAccountName, state.AlloraStakingAccountName, rewardCoins)
```

Comment:
not sure if I got this right: apparently you're double spending here. Adding stake to the account and sending the same amount to the AlloraStakingAccount

Reply:
one operation sends the tokens, the other increments the data structures that track their proportion of total stake

Reply:
AddStake doesn't actually move tokens itself